frontend/src/components/common/form/FastDatePicker.tsx

```diff
@@ -7,6 +7,7 @@ import { formikFieldMemo } from 'utils';
 import classNames from 'classnames';
 import GenericModal from '../GenericModal';
 import { appraisalDateWarning } from 'features/projects/common';
+import * as Popper from 'popper.js';
```

Comment:
googled this to see what it was, that's super handy! also the website is cute lol